tests/MessagesTest.elm

```diff
@@ -0,0 +1,240 @@
+                            , comment =
+                                Just
+                                    ( anotherRange
+                                    , "{-|A function.\n-}"
+                                    )
+                            , exposed = True
+                            }
+                     in
+                     Messages.entityToString entity True
+                    )
+        , Test.test "Test 3." <|
+            \() ->
+                Expect.equal
+                    ("line 13, function with parameters (aString, anInt) "
+                        ++ "\"someFunction\" with comment \"{-|A function with "
+                        ++ "parameters.\n-}\""
+                    )
+                    (let
+                        entity =
+                            { range = aRange
+                            , eType = Function [ "aString", "anInt" ]
+                            , name = "someFunction"
+                            , comment =
+                                Just
+                                    ( anotherRange
+                                    , "{-|A function with parameters.\n-}"
+                                    )
+                            , exposed = False
+                            }
+                     in
+                     Messages.entityToString entity True
+                    )
+        , Test.test "Test 4." <|
+            \() ->
+                Expect.equal
+                    ("line 13, exposed record with fields (aMap, aListOfInts) "
+                        ++ "\"SomeRecord\" with comment \"{-|A record.\n-}\""
+                    )
+                    (let
+                        entity =
+                            { range = aRange
+                            , eType = Record [ "aMap", "aListOfInts" ]
+                            , name = "SomeRecord"
+                            , comment =
+                                Just ( anotherRange, "{-|A record.\n-}" )
+                            , exposed = True
+                            }
+                     in
+                     Messages.entityToString entity True
+                    )
+        , Test.test "Test 5." <|
+            \() ->
+                Expect.equal
+                    "line 13, exposed type definition \"SomeType\""
+                    (let
+                        entity =
+                            { range = aRange
+                            , eType = TypeDef
+                            , name = "SomeType"
+                            , comment = Nothing
+                            , exposed = True
+                            }
+                     in
+                     Messages.entityToString entity False
+                    )
+        ]
+
+
+{-| Tests the issueToString function.
+-}
+issueToString : Test.Test
+issueToString =
+    Test.describe "Test the issueToString function."
+        [ Test.test "Test 1." <|
```

Comment:
Rather then enumerating the tests ("Test 1", "Test 2" etc.), please describe what is actually tested. For example, "Detects missing top-level module comment".

Please describe all the other tests in the repo.